src/org/traccar/protocol/Tk103ProtocolDecoder.java

```diff
@@ -41,14 +41,14 @@ public Tk103ProtocolDecoder(Tk103Protocol protocol) {
             .number("(d+)(,)?")                  // device id
             .expression(".{4},?")                // command
             .number("d*")                        // imei?
-            .number("(dd)(dd)(dd),?")            // date
+            .number("(dd)(dd)(dd),?")            // date (yymmdd or mmddyy?)
```

Comment:
Are you sure it is MMDDYY and not DDMMYY? I think we need to clarify it and remove question mark.